Fails to build due to pulumi deps

```bash
go version
go version go1.11.1 darwin/amd64

env | grep GO
GOPATH=/Users/leigh.schrandt/go
GO111MODULE=on

go get -u github.com/pulumi/kubespy
go: finding github.com/codahale/hdrhistogram latest
go: finding 9fans.net/go latest
go: finding github.com/alcortesm/tgz latest
go: finding github.com/google/btree latest
go: finding k8s.io/kube-openapi latest
go: finding golang.org/x/arch latest
go: finding github.com/golang/glog latest
go: finding github.com/mdempsky/gocode latest
go: finding github.com/docker/spdystream latest
go: finding github.com/reconquest/loreley latest
go: finding github.com/prometheus/procfs latest
go: finding github.com/jbenet/go-context latest
go: finding k8s.io/utils latest
go: finding github.com/kevinburke/ssh_config latest
go: finding github.com/google/gofuzz latest
go: finding github.com/gosuri/uilive latest
go: finding github.com/beorn7/perks latest
go: finding github.com/mxk/go-flowrate latest
go: finding github.com/coreos/pkg latest
go: finding github.com/emicklei/go-restful-swagger12 latest
go: finding github.com/k0kubun/colorstring latest
go: finding github.com/mitchellh/go-ps latest
go: finding golang.org/x/tools latest
go: finding github.com/flynn/go-shlex latest
go: finding github.com/fatih/gomodifytags latest
go: finding github.com/skratchdot/open-golang latest
go: finding k8s.io/apiextensions-apiserver latest
go: finding k8s.io/apiserver latest
go: finding golang.org/x/sys latest
go: finding github.com/golang/groupcache latest
go: finding github.com/mailru/easyjson latest
go: finding github.com/golang/lint latest
go: finding github.com/xiang90/probing latest
go: finding github.com/cosiner/argv latest
go: finding golang.org/x/oauth2 latest
go: finding golang.org/x/sync latest
go: finding golang.org/x/lint latest
go: finding github.com/anmitsu/go-shlex latest
go: finding google.golang.org/genproto latest
go: finding github.com/stevvooe/resumable latest
go: finding github.com/PuerkitoBio/urlesc latest
go: finding github.com/ugorji/go/codec latest
go: finding k8s.io/api latest
go: finding honnef.co/go/tools latest
go: finding github.com/gregjones/httpcache latest
go: finding golang.org/x/net latest
go: finding github.com/gophercloud/gophercloud latest
go: finding github.com/howeyc/gopass latest
go: finding github.com/pquerna/cachecontrol latest
go: finding github.com/davidrjenni/reftools latest
go: finding github.com/sqs/goreturns latest
go: finding github.com/josharian/impl latest
go: finding github.com/tmc/grpc-websocket-proxy latest
go: finding github.com/yudai/golcs latest
go: finding golang.org/x/time latest
go: finding github.com/prometheus/client_model latest
go: finding github.com/texttheater/golang-levenshtein latest
go: finding github.com/modern-go/concurrent latest
go: finding golang.org/x/crypto latest
go: finding gopkg.in/tomb.v1 latest
go: finding k8s.io/apimachinery latest
go: finding gopkg.in/check.v1 latest
go: finding github.com/ramya-rao-a/go-outline latest
go: finding github.com/acroca/go-symbols latest
go: finding github.com/coreos/go-systemd latest
go: finding github.com/prometheus/common latest
# k8s.io/client-go/plugin/pkg/client/auth/azure
../../../go/pkg/mod/k8s.io/client-go@v9.0.0+incompatible/plugin/pkg/client/auth/azure/azure.go:246:4: cannot use expiresIn (type string) as type json.Number in field value
../../../go/pkg/mod/k8s.io/client-go@v9.0.0+incompatible/plugin/pkg/client/auth/azure/azure.go:247:4: cannot use expiresOn (type string) as type json.Number in field value
../../../go/pkg/mod/k8s.io/client-go@v9.0.0+incompatible/plugin/pkg/client/auth/azure/azure.go:248:4: cannot use expiresOn (type string) as type json.Number in field value
../../../go/pkg/mod/k8s.io/client-go@v9.0.0+incompatible/plugin/pkg/client/auth/azure/azure.go:265:23: cannot use token.token.ExpiresIn (type json.Number) as type string in assignment
../../../go/pkg/mod/k8s.io/client-go@v9.0.0+incompatible/plugin/pkg/client/auth/azure/azure.go:266:23: cannot use token.token.ExpiresOn (type json.Number) as type string in assignment
# github.com/pulumi/pulumi-kubernetes/pkg/client
../../../go/pkg/mod/github.com/pulumi/pulumi-kubernetes@v0.17.3/pkg/client/util.go:72:7: undefined: dynamic.ClientPool
../../../go/pkg/mod/github.com/pulumi/pulumi-kubernetes@v0.17.3/pkg/client/util.go:85:7: undefined: dynamic.ClientPool

I'm seeing the same...

```
# k8s.io/client-go/plugin/pkg/client/auth/azure
../../../go/pkg/mod/k8s.io/client-go@v9.0.0+incompatible/plugin/pkg/client/auth/azure/azure.go:246:4: cannot use expiresIn (type string) as type json.Number in field value
../../../go/pkg/mod/k8s.io/client-go@v9.0.0+incompatible/plugin/pkg/client/auth/azure/azure.go:247:4: cannot use expiresOn (type string) as type json.Number in field value
../../../go/pkg/mod/k8s.io/client-go@v9.0.0+incompatible/plugin/pkg/client/auth/azure/azure.go:248:4: cannot use expiresOn (type string) as type json.Number in field value
../../../go/pkg/mod/k8s.io/client-go@v9.0.0+incompatible/plugin/pkg/client/auth/azure/azure.go:265:23: cannot use token.token.ExpiresIn (type json.Number) as type string in assignment
../../../go/pkg/mod/k8s.io/client-go@v9.0.0+incompatible/plugin/pkg/client/auth/azure/azure.go:266:23: cannot use token.token.ExpiresOn (type json.Number) as type string in assignment
# github.com/pulumi/pulumi-kubernetes/pkg/client
../../../go/pkg/mod/github.com/pulumi/pulumi-kubernetes@v0.17.3/pkg/client/util.go:72:7: undefined: dynamic.ClientPool
../../../go/pkg/mod/github.com/pulumi/pulumi-kubernetes@v0.17.3/pkg/client/util.go:85:7: undefined: dynamic.ClientPool
```
with `go version go1.11.1 darwin/amd64`

--- a/README.md
+++ b/README.md
@@ -41,10 +41,11 @@ Prerequisite: [Go](https://golang.org/) version 1.11 or later
 kubespy can be installed via golang by running the following:
 
 ```sh
-export GO111MODULE=on
-
-# If $GOBIN is not on your path, you'll need to install the library elsewhere.
-go get -u github.com/pulumi/kubespy
+git clone git@github.com:pulumi/kubespy.git
+cd kubespy
+# If $GOPATH is not set, you can run `go build` instead, and then put in a
+# binary directory on your path
+GO111MODULE=on go install
 ```
 
 ## Usage

